.changeset/calm-olives-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/website': patch
+---
+
+Made tables' last column wrap. Those are typically used for long descriptions.
```

Comment:
```suggestion
Updated API documentation. 
```

We don't really need a changeset for the website but if we want the package readme files to update NPM docs we should include a patch for each of the readmes changed. Otherwise the NMP docs only get updated on the next release.

For minor doc changes I'm usually ok with that but if you feel npm docs should be updated include the packages as a patch release.

Reply:
Therefore the update message should say what changed in the packages (not the formatting on the website)